### PR TITLE
Add jansky and solar flux unit

### PR DESCRIPTION
**Pull request by triallax**

### CODE REVIEW

File: numbat/modules/extra/astronomy.nbt [hunk 1]
```diff
@@ -23,3 +23,15 @@ unit solar_radius: Length = 6.957e5 km
 
 dimension RadiantFlux = Power
 unit solar_luminosity: RadiantFlux = 3.828e26 W
+
+dimension SpectralFluxDensity = RadiantFlux / Area / Frequency
+
+@name("Jansky")
+@url("https://en.wikipedia.org/wiki/Jansky")
+@aliases(janskys, Jy: short)
+unit jansky: SpectralFluxDensity = 10e-26 W / m^2 / Hz
```
Comment (sharkdp) [inline numbat/modules/extra/astronomy.nbt:32]: 10^-26 is 1e-26, not 10e-26.

```suggestion
unit jansky: SpectralFluxDensity = 1e-26 W / m^2 / Hz
```

Comment (triallax) [reply numbat/modules/extra/astronomy.nbt:32]: oops

Comment (triallax) [reply numbat/modules/extra/astronomy.nbt]: thanks for catching this

**Review by sharkdp — APPROVED** [review]
Thank you

**Pull request merged**